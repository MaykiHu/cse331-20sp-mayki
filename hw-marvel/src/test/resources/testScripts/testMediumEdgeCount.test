#Tests for the edgeCount of a medium dataset, from mediumSizeSuperheroes.tsv
#mediumSizeSuperheroes.tsv should be a valid file under the marvel data folder

LoadGraph mediumHeroes mediumSizeSuperheroes.tsv
GetEdgeCount mediumHeroes