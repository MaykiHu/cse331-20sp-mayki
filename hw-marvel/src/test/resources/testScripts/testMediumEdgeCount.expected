#Tests for the edgeCount of a medium dataset, from mediumSizeSuperheroes.tsv
#mediumSizeSuperheroes.tsv should be a valid file under the marvel data folder

loaded graph mediumHeroes
Number of edges in mediumHeroes is: 96